app/AppKernel.php

```diff
@@ -50,7 +50,7 @@ public function registerBundles()
             new Solution\MongoAggregationBundle\SolutionMongoAggregationBundle(),
         );
 
-        if (in_array($this->getEnvironment(), array('dev', 'test'))) {
+        if (in_array($this->getEnvironment(), array('dev', 'test', 'behat'))) {
```

Comment:
As the web_profiler toolbar is disabled, i would suggest to only use the production bundles